qiita_db/test/test_parameters.py

```diff
@@ -15,6 +15,29 @@
 @qiita_test_checker()
 class PreprocessedIlluminaParamsTests(TestCase):
 
+    def test_create(self):
+        obs_obj = PreprocessedIlluminaParams.create(
```

Comment:
The actual code isn't casting the types and is only passing in strings, can the test be updated to reflect that?